src/js/components/sharedComponents/CheckboxTree.jsx

```diff
@@ -17,6 +17,8 @@ import 'react-checkbox-tree/lib/react-checkbox-tree.css';
 const propTypes = {
     data: PropTypes.array,
     isLoading: PropTypes.bool,
+    isError: PropTypes.bool,
```

Comment:
The CheckboxTree should not be rendering error messages or loading or no results unless those messages are going to be displayed in the checkbox tree.  The CheckboxTree should be rendering a checkbox tree. The loading, error and no results should be handled outside of this component.